hipster-core/src/main/java/es/usc/citius/hipster/algorithm/IDAStar.java

```diff
@@ -40,39 +37,18 @@
  * @param <C> comparable cost used to compare states.
  * @param <N> type of the heuristic search node.
  *
- * @author Pablo Rodríguez Mier <<a href="mailto:pablo.rodriguez.mier@usc.es">pablo.rodriguez.mier@usc.es</a>>
```

Comment:
As the implementation is not completely yours, it is better to maintain the original authors, you can add yourself below. Also, it is ok if you want to specify an email address for contact, although not required.